File: ADP.LookupServices/Lookup.Services/DTOsAndModels/VehicleLookup/PaintThicknessDTO.cs
using ShiftSoftware.ADP.Models;
using System.Collections.Generic;

namespace ShiftSoftware.ADP.Lookup.Services.DTOsAndModels.VehicleLookup;

[TypeScriptModel]
public class PaintThicknessDTO
{
    public IEnumerable<PaintThicknessPartDTO> Parts { get; set; }

    public IEnumerable<PaintThicknessImageDTO> ImageGroups { get; set; } = new List<PaintThicknessImageDTO>();
}

File: ADP.WebComponents/adp-web-components/src/global/types/vehicleLookup/paintThicknessDTO.ts
import type { paintThicknessPartDTO } from './paintThicknessPartDTO';
import type { paintThicknessImageDTO } from './paintThicknessImageDTO';
export type paintThicknessDTO = {
    parts: paintThicknessPartDTO[];
    imageGroups: paintThicknessImageDTO[];
};